src/_typography.scss

```diff
@@ -38,6 +38,10 @@ label.h4,
     opacity: 0.75;
 }
 
+label:disabled {
+    opacity: 0.5;
```

Comment:
I feel this gives less contrast for an insensitive label under Greenfield... maybe 0.66 at least? :)